ajax/libs/jqgrid/package.json

```diff
@@ -1,21 +1,48 @@
 {
   "name": "jqgrid",
   "filename": "js/jquery.jqGrid.min.js",
-  "version": "4.6.0",
+  "version": "5.0.1",
   "description": "jqGrid is an Ajax-enabled JavaScript control that provides solutions for representing and manipulating tabular data on the web.",
   "homepage": "http://www.trirand.com/blog/",
   "keywords": [
+    "jQuery",
+    "jqGrid",
+    "JavaScript",
     "grid",
-    "jquery"
+    "table",
+    "pivot",
+    "tree",
+    "subgrid",
+    "sorting",
+    "paging",
+    "editing",
+    "grouping",
+    "searching",
+    "ajax"
   ],
-  "license": "CC-BY-NC-3.0",
+  "license": "Creative Commons Attribution-Non-Commercial 3.0 License",
   "repository": {
     "type": "git",
-    "url": "https://github.com/tonytomov/jqGrid"
+    "url": "https://github.com/tonytomov/jqGrid/tree/master"
```

Comment:
May I ask that why the repo url need to add `/tree/master`?
Because the origin url is as same with `https://github.com/tonytomov/jqGrid/tree/master`.
I don't know is it necessary to change the url.
Thank you.